activation/interface.go

```diff
@@ -80,8 +80,8 @@ type layerClock interface {
 }
 
 type nipostBuilder interface {
-	BuildNIPost(ctx context.Context, sig *signing.EdSigner, challenge *types.NIPostChallenge) (*nipost.NIPostState, error)
-	Proof(ctx context.Context, nodeID types.NodeID, challenge []byte) (*types.Post, *types.PostInfo, error)
+	BuildNIPost(context.Context, *signing.EdSigner, *types.NIPostChallenge) (*nipost.NIPostState, error)
+	Proof(context.Context, types.NodeID, []byte) (*types.Post, *types.PostInfo, error)
```

Comment:
I'm not sure about this. Previously it was easy to understand what the 3rd parameter of `Proof()` is by looking at its name. Now, it's impossible.

Reply:
Yeah I wasn't sure how to best fix this. I guess line breaks after commas work too